src/Command.php

```diff
@@ -162,7 +162,7 @@ public function getExecCommand()
     {
         if ($this->_execCommand===null) {
             $command = $this->getCommand();
-            if (!$command) {
+            if (!$command || !is_executable($command)) {
```

Comment:
is_executable() was your own suggestion from earlier.
"whereis" is present in OSX, and all Linux variants that I can think of. Hence, the isWindows check... It is an assumption, but generally a safe one. Though if there is a better suggestion, I'd be keen to help figure this out.